### PR TITLE
Tools: size-limit

### PR DESCRIPTION
**Pull request by Drapegnik**
https://github.com/ai/size-limit

### CODE REVIEW

**Comment by Drapegnik** [comment]
[replace bundlesize with size-limit](https://trello.com/c/lW23vKh2/459-replace-bundlesize-with-size-limit)

**Comment by Drapegnik** [comment]
@UladBohdan, to test GitHub action we need to merge new code to develop

File: package.json [hunk 1]
```diff
@@ -28,8 +28,8 @@
     "postinstall": "git submodule update --init && npm run fetch-data",
     "semantic-release": "semantic-release",
     "travis-deploy-once": "travis-deploy-once",
-    "bundlesize": "bundlesize",
-    "size": "npm run reset-cache && npm run build && npm run bundlesize",
+    "size-build": "npm run reset-cache && npm run build",
+    "size-limit": "size-limit",
```
Comment (UladBohdan) [inline package.json:32]: why not to make `size-build` a part of `size-limit` script? it does not seem to be functional without it

File: Makefile [hunk 2]
```diff
@@ -17,4 +17,5 @@ update:
 	npm run fetch-data
 
 build-size:
-	npm run size
+	npm run build-size
```
Comment (UladBohdan) [inline Makefile]: `build-size` here but `size-build` in package.json

**Review by UladBohdan — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Drapegnik) [reply package.json:32]: https://github.com/andresz1/size-limit-action#usage

File: Makefile (hunk 2 above)
Comment (Drapegnik) [reply Makefile]: this is for `make s + <TAB> -> make start` as more used case :)

**Review by UladBohdan — APPROVED** [review]

**Pull request merged**